nautobot/core/tests/test_navigations.py

```diff
@@ -46,6 +46,8 @@ def test_menu_item_attributes(self):
                                     expected_name = "Change Log"
                                 elif expected_name == "Controller Managed Device Groups":
                                     expected_name = "Managed Device Groups"
+                                elif expected_name == "Object Metadatas":
+                                    expected_name = "Object Metadata"
```

Comment:
Would it be possible/more correct to fix this by updating `ObjectMetadata.verbose_name_plural`?